File: src/targets/java/okhttp/fixtures/short.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
<<<<<<< HEAD
  .url("https://httpbin.org/anything")
=======
  .url("http://httpbin.org/anything")
>>>>>>> upstream/master
  .get()
  .build();

Response response = client.newCall(request).execute();

File: src/targets/kotlin/okhttp/fixtures/short.kt
val client = OkHttpClient()

val request = Request.Builder()
<<<<<<< HEAD
  .url("https://httpbin.org/anything")
=======
  .url("http://mockbin.com/har")
>>>>>>> upstream/master
  .get()
  .build()

val response = client.newCall(request).execute()